types/src/front/lib/core_api.ts

```diff
@@ -152,7 +153,9 @@ export class CoreAPI {
   ) {
     this._url = config.url;
     this._logger = logger;
+    this._apiKey = process.env.CORE_API_KEY || null;
```

Comment:
We want to avoid process in types

We must pass it to the Core API and use config in front 👍

Reply:
☝️

Reply:
`core_api.ts` cannot be used from the client though